app/src/main/res/menu/bottom_menu.xml

```diff
@@ -12,6 +12,10 @@
     android:id="@+id/search"
     android:title="Search"
     android:icon="@drawable/ic_search"/>
+  <item
+      android:id="@+id/my_reviews"
+      android:title="My"

```

Comment:
Would not `Mine` be better ?

Reply:
I had a lot of doubts about that and wanted to put "My Reviews" first but that was too long... I also thought about "Mine" but it looked a little bit weird in the app so I just kept "My" and will change it later if we find something nicer.